tests/test_rundb.py

```diff
@@ -0,0 +1,45 @@
+import unittest
```

Comment:
Please add standard header and also make sure that this file gets run by the normal `cylc test-battery`.

Reply:
Oh, I checked a test in another branch for the header (remembered having to add the header in isodatetime), but that file didn't have the header... turns out I forgot to include the headers in the coverage pull request. So this will save some time for the reviewers of the other pull request. Thanks @matthewrmshin !

Updated header, and moved the test to `lib/cylc/tests`, where the other unit tests are being moved (see #2766). The easiest way to run this tests right now outside of an IDE is 

```
# from the project root folder
# just the test
PYTHONPATH=lib py.test lib/cylc/tests/

# coverage:
PYTHONPATH=lib py.test --cov=lib/cylc lib/cylc/tests/ --cov-append
```

The coverage pull request should start running the tests with Travis (will double-check tomorrow that that's working). And another pull request to add `setup.py` in Cylc will take care of running the tests via `setup.py`, so that that works the same for developers and in Travis-CI :+1: